### DISCUSSION TITLE
Big integer literals

### DISCUSSION THREAD

**Discussion by pu0238** [nuxt/nuxt#18816] [Questions]
### Environment

- Operating System: `Linux`
- Node Version:     `v16.15.1`
- Nuxt Version:     `3.0.0`
- Nitro Version:    `1.0.0`
- Package Manager:  `npm@9.2.0`
- Builder:          `vite`
- User Config:      `app`, `components`, `modules`, `nitro`
- Runtime Modules:  `@nuxtjs/tailwindcss@6.2.0`
- Build Modules:    `-`

### Describe the bug

I can't use the above lib, by the following error:

Big integer literals are not available in the configured target environment ("chrome87", "edge88", "es2020", "firefox78", "safari13" + 2 overrides)

### Logs

```shell-script
✘ [ERROR] Big integer literals are not available in the configured target environment ("chrome87", "edge88", "es2020", "firefox78", "safari13" + 2 overrides)

    node_modules/@noble/ed25519/lib/index.js:191:46:
      191 │         let p = this.multiplyUnsafe(CURVE.l / 2n).double();
```

**Comment by TheAlexLichter** [comment]
Hey! Could you kindly provide a [reproduction](https://nuxt.com/docs/community/reporting-bugs/#create-a-minimal-reproduction)? 🙏
If you want based on [my stackblitz](https://stackblitz.com/edit/nuxtdiscussions18816?file=nuxt.config.ts,app.vue)

Related: https://github.com/nuxt/nuxt/issues/14601

**Reply by pu0238** [reply]
I was using nuxt at version 3.0.0. Given reproduction was on  3.1.2 so i upgraded everything:
https://stackblitz.com/edit/nuxtdiscussions18816-a4knmk?file=nuxt.config.ts

**Reply by pu0238** [reply]
on version 3.1.2 i got error:

bigint: Failed to load bindings, pure JS will be used (try npm run rebuild?)

**Reply by pu0238** [reply]
without error it should be something like this 
![image](https://user-images.githubusercontent.com/90582259/217035150-261aff45-af53-4ed0-8fd7-03ebbcec821e.png)
